index.html

```diff
@@ -30,26 +30,29 @@ <h1>I'm Redwan.<br/>
                 </p>
             </div>
             <!-- End of header top -->
-
             <div class="headerBottom">
                 <a href="#">LET’S CONNECT</a>
                 <div class="link-btn">
-                    <img src="images/Twitter.svg" alt="Twitter"/>
-                    <span>
-                        <img src="images/Instagram.png" alt="Instagram"/>
-                        <img src="Images/VectorN.png" alt="Instagram"/>
-                    </span>
-                    
-                    <img src="images/link-Vector3.svg" alt=""/>
-                    <img src="images/link-Vector4.svg" alt=""/>
-                    <img src="images/link-Vector5.svg" alt=""/>
+                    <ul>
```

Comment:
- [ ] Great job fixing the gap between the social icons and wrapping the icons list in an unordered list. But using the ul tag comes with a margin-left and in the template, all contents of the headline section are aligned on the left. Kindly remove the default margin and padding-left so that the list of social icons can also be aligned with other contents of the headline section.
### your design
![image](https://github.com/ethred/Portfolio/assets/94551484/e25dff03-b7c3-49b3-8c8e-691f1ce06f88)
### Figma
![image](https://github.com/ethred/Portfolio/assets/94551484/136a85ee-f323-4e62-bc64-fee7d86b128a)